### ISSUE TITLE
npm audit issues prevent install

### ISSUE THREAD

**Issue by AleXSR700** [sakai-to/node-red-contrib-spreadsheet-in#16]
Unfortunately, the package can not longer be installed.

```
# npm audit report

xlsx  *
Severity: high
Prototype Pollution in sheetJS - https://github.com/advisories/GHSA-4r6h-8v6p-xvw6
No fix available
node_modules/xlsx
  node-red-contrib-spreadsheet-in  *
  Depends on vulnerable versions of xlsx
  node_modules/node-red-contrib-spreadsheet-in

2 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.
```
Any chance for a fix?

**Comment by sakai-to** [comment]
@AleXSR700 Thank you for the report. I have changed the xlsx version to 0.19.3. Unfortunately, I cannot publish my new package to npm at present because of my npm user account issue. I will try to publish it in a few days when the issue is resolved.

**Comment by sakai-to** [comment]
I published the new version 0.7.1 to resolve this issue.
